### PR TITLE
Create bi-nif.json

### PR DESCRIPTION
**Pull request by andreaszenasidi**

### CODE REVIEW

File: lists/bi/bi-nif.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  "sector": [],
+  "code": "BI-NIF",
+  "confirmed": false,
+  "deprecated": false,
+  "listType": "primary",
+  "access": {
+    "availableOnline": true,
+    "onlineAccessDetails": null,
+    "publicDatabase": "https://www.obr.bi/index.php/en/verification-de-validite-de-documents?option=com_odvc&view=odvc&id=1&type_doc=4",
+    "guidanceOnLocatingIds": "The NIF number needs to be requested from the company or organisation. After obtaining the NIF number, the publicly available web page can be used to verify the ownership and validity of the NIF number. On the web page, for Type de document select 'NIF contribuable' and enter the NIF number in the displayed textbox. The result returns a plain text stating who the NIF belongs to and whether the number is recognised by the OBR.",
```
Comment (kd-ods) [inline lists/bi/bi-nif.json:29]: The same typo as above appears here.

**Review by kd-ods — CHANGES_REQUESTED** [review]
Just a few minor things to iron out. Otherwise looks good.

File: lists/bi/bi-nif.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    "charity",
+    "company",
+    "unincorporated_body",
+    "trust"
+  ],
+  "sector": [],
+  "code": "BI-NIF",
+  "confirmed": false,
+  "deprecated": false,
+  "listType": "primary",
```
[the remark this answers is not in the document]
Comment (siwhitehouse) [reply lists/bi/bi-nif.json:24]: Yes, they should be secondary as per the description in the schema: "[Examples of secondary lists are VAT or Tax identifiers, or employer identification numbers](https://github.com/org-id/register/blob/master/schema/codelist-listType.json#L19)"

**Comment by kd-ods** [comment]
@andreaszenasidi  - can you just confirm that you've made all the changes you want to make and that you're happy for us to re-review this now? (There's a 're-request review' button that you can use by our names in the 'Reviewers' section on the right-hand side.)

**Comment by andreaszenasidi** [comment]
@kd-ods  Oh sorry, I was not aware of these steps. I confirm I made all the requested changes.

**Review by kd-ods — APPROVED** [review]
Looks good. Thanks, Andrea. I'll merge and update the org-id website.

**Pull request merged**